restapi/configure_deathstar.go

```diff
@@ -41,14 +46,11 @@ var info = `{
 	"starship_class": "Deep Space Mobile Battlestation",
 	"api": [
 		"GET   /v1",
-		"GET   /v1/healthz",
 		"POST  /v1/request-landing",
-		"PUT   /v1/cargobay",
-		"GET   /v1/hyper-matter-reactor/status",
```

Comment:
why were these deleted?

Reply:
It seems these endpoints were never implemented, so if we're going to use this `/v1/` info interface (which we didn't so far), it's probably better to remove them.

I'm happy to make a separate PR for it if you prefer though.

Reply:
This `info` is used for any GETs performed to the server. We should keep them since it looks good on demos even if they are not implemented.

Reply:
OK then, I'll keep them!